pageserver/src/tenant/size.rs

```diff
@@ -87,6 +87,9 @@ impl SegmentMeta {
             LsnKind::BranchPoint => true,
             LsnKind::GcCutOff => true,
             LsnKind::BranchEnd => false,
+            LsnKind::LeasePoint => true,
+            LsnKind::LeaseStart => false,
```

Comment:
Noting that this is different than BranchStart, but the lsn will be the same as LeasePoint.

Is that because we cannot ancestorless branches? Maybe we'll need to have a call about this :)

Reply:
It is valid that this is different from BranchStart because we can never have the gc_cutoff move past the "lease point" in the way we can with branches.